src/app/components/detailed-page/components/detailed-page-hero/detailed-page-hero.component.ts

```diff
@@ -9,6 +9,7 @@ export class DetailedPageHeroComponent {
   @Input() public title?: string;
   @Input() public img?: string | null;
   @Input() public description?: string;
+  @Input() public isPersonPage = false;
```

Comment:
Are yoku sure we need this check? Probably we create to separte pages. Or am I wrong?

Reply:
Person pages uses  `detailed-page-hero` with same layout, same inputs etc. as other detailed pages, but to place `known for` block correctly in layout, had to add this input to apply modifier class to grid inside `<detailed-page-hero>`. 
Making separate component is an option here, but it will be 95% the same as the original one, except 5% of changes that will be css styles to adjust layout of the component, so i'm not sure if it's good solution here.